ingest/expression_files/dense_ingestor.py

```diff
@@ -28,6 +28,8 @@
 class DenseIngestor(GeneExpression, IngestFiles):
     # ToDo SCP-2635
     ALLOWED_FILE_TYPES = ["text/csv", "text/plain", "text/tab-separated-values"]
+    GENE_BATCH_SIZE = 5
```

Comment:
Out of curiosity, why is `GENE_BATCH_SIZE` so much lower than `DATA_ARRAY_BATCH_SIZE` when the `Gene` documents are so much smaller overall compared to `DataArray` documents?

Reply:
Great question -- that limit on 5 genes was there from before.  I suspect that was a limit we used as a quick-and-dirty batching size when we made the first proof-of-concept of this ingestor, and then just stuck around...